android/app/src/main/java/com/zooniversemobile/MainActivity.java

```diff
@@ -87,6 +97,29 @@ private boolean playServicesAvailable() {
         return true;
     }
 
-
-
+    private void sendNotification(String messageTitle, String messageBody, String projectID) {
+        Intent intent = new Intent(this, MainActivity.class);
```

Comment:
Maybe choose a different name for your Intent here? `Intent intent` just seems a bit odd semantically.